test/integration/crud/find.test.js

```diff
@@ -1,13 +1,18 @@
 'use strict';
-const { assert: test, setupDatabase } = require('../shared');
+const { assert: test } = require('../shared');
 const { expect } = require('chai');
 const sinon = require('sinon');
 const { setTimeout } = require('timers');
 const { Code, ObjectId, Long, Binary, ReturnDocument } = require('../../../src');
 
 describe('Find', function () {
-  before(function () {
-    return setupDatabase(this.configuration);
+  let client;
```

Comment:
introducing mocha hooks cus there was a close leak, classic callback didn't reach the end stuff

Reply:
Why did these leaks only show up now?  We didn't change any codepaths so I would expect no change in test behavior.